### PR TITLE
MWPW-161872: Self host marketo's forms2.min.js

### PR DESCRIPTION
**Pull request by JasonHowellSlavin**
* Adds forms2.min.js to deps
* Modifies marketo.js block code to use new dependency
* Updates workflow for IMS to also check for form2 updates

Resolves: [MWPW-161872](https://jira.corp.adobe.com/browse/MWPW-161872)

Context: Form at the bottom of the page

**Test URLs:**

- Before: https://main--milo--jasonhowellslavin.hlx.live/drafts/slavin/bacom-pages/
- After: https://self-host-marketo-forms-js--milo--jasonhowellslavin.hlx.live/drafts/slavin/bacom-pages/

### CODE REVIEW

File: .github/workflows/update-dependencies.yaml [hunk 1]
```diff
@@ -26,5 +26,18 @@ jobs:
...
           })
+    - name: Update forms2 and create PR if needed
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const updateDependency = require('./.github/workflows/update-script.js')
+          updateDependency({
+            github, 
+            context, 
+            title: '[AUTOMATED-PR] Update froms2.min.js dependency',
```
Comment (meganthecoder) [inline .github/workflows/update-dependencies.yaml:37]: Typo
```suggestion
            title: '[AUTOMATED-PR] Update forms2.min.js dependency', 
```

**Review by meganthecoder — COMMENTED** [review]
I'm seeing forms2 being loaded twice, once from libs and once from engage

File: .github/workflows/update-dependencies.yaml (hunk 1 above)
Comment (JasonHowellSlavin) [reply .github/workflows/update-dependencies.yaml]: Great catch, thank you!

**Comment by Brandon32** [comment]
> I'm seeing forms2 being loaded twice, once from libs and once from engage

The second forms2 is coming from the XDFrame for cross-domain posts. 
It looks like the form does not submit, we need to make sure it posts correctly.

**Review by mokimo — APPROVED** [review]
Cool to see this action expanding and re-used!

**Review by Brandon32 — APPROVED** [review]

**Comment by JasonHowellSlavin** [comment]
> I'm seeing forms2 being loaded twice, once from libs and once from engage

This is expected. Part of CORS functionality sent by Marketo.

**Comment by JasonHowellSlavin** [comment]
@mokimo @Brandon32 @meganthecoder updated the `getPrDescription` function as Brandon pointed out it was built with IMS in mind. Please take a look! Thank you.

**Review by meganthecoder — APPROVED** [review]

**Review by mokimo — APPROVED** [review]

File: libs/blocks/marketo/marketo.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ import {
   loadLink,
   localizeLink,
   createTag,
+  getConfig,
```
Comment (Brandon32) [inline libs/blocks/marketo/marketo.js:22]: We will need to call `setConfig` in the unit tests and update the import mapped marketo-utils.js.

**Pull request merged**